src/main/kotlin/com/terraformation/backend/accelerator/ApplicationVariableValuesFetcher.kt

```diff
@@ -33,6 +34,7 @@ class PreScreenVariableValuesFetcher(
     const val STABLE_ID_NUM_SPECIES = "22"
     const val STABLE_ID_PROJECT_TYPE = "3"
     const val STABLE_ID_TOTAL_EXPANSION_POTENTIAL = "24"
+    const val STABLE_ID_WEBSITE = "27"
```

Comment:
Oops, this should be part of the next PR in the stack, not this one. I'll move it there before merging.